sdk/src/test/java8/com/google/cloud/dataflow/sdk/options/PipelineOptionsFactoryJava8Test.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2016 Google Inc.
```

Comment:
Clarification: probably requires a change to `checkstyle.xml` rules, etc. The change should be done atomically across all files.